template-for-repository/proofs/Makefile-project-defines

```diff
@@ -13,6 +13,12 @@
 # Makefile.common.
 ################################################################
 
+# Name of this proof project, displayed in proof reports
```

Comment:
What the name of this proof project?  Do you mean "Amazon FreeRTOS" or "s2n"?  I'd give an example.  

In the development branch of the aws/aws-iot-device-sdk-embedded-C, the starter kit it submoduled under tools, and the two projects libraries/standard/mqtt and libraries/standard/http have independent subdirectories cbmc/proofs with independent Makefile-project-defines.  Yet CI has no trouble finding all of these proofs and running them all together.  Are you expecting that there is a single top-level Makefile-project-defines for the whole source tree?